README.md

```diff
@@ -1,99 +1,165 @@
 
 - Ruby, RubyGems
 - Sass (`sudo gem install sass`)
-- Node.js (10.14.2 or higher is recommended) and NPM (see <https://nodejs.org/en/download/package-manager/>)
-- Go to project root, then run `npm install`
+- Node.js (12.22.3 is recommended)
+- NPM (see <https://nodejs.org/en/download/package-manager/>)
 - Grunt (`sudo npm install -g grunt-cli`)
 
-### Use a custom domain
-In order to use your custom domain, please put it in a file named `CNAME` inside `scripts` folder of your local clone of binary-static.
+## Quickstart
+
+1. **Create a new organization**
+   Create a new organization in [github.com](https://github.com/account/organizations/new?coupon=&plan=team_free)
+
+2. **Fork the project**
+
+   In order to work on your own version of the SmartTrader application, please fork the project in your newly created organization.
+
+3. **Clone using SSH**
+
+   ```sh
+   git clone git@github.com:<organization_name>/binary-static.git smart-trader
+   ```
+
+4. **Enter project directory**
+
+   ```sh
+   cd smart-trader
+   ```
+
+5. **Add DSmartTrader project as your upstream**
+
+   ```sh
+   git remote add upstream git@github.com:deriv-com/binary-static.git
+   ```
+
+6. **Install your dependencies:**
+
+   ```sh
+     npm ci
+   ```
+
+7. **To start developing:**
+
+   ```sh
+   npm run start
+   ```
+
+8. **Open the source code and start editing!**
+
+   Your site is now running at [https://localhost:443](https://localhost:443)
+
+## Preview on your local machine
+
+- To preview your changes locally, run `sudo grunt serve`
+- It will watch for Javascript or CSS changes and will rebuild on every change you make.
+- To test changes made to templates, you need to re-compile them:
+  - `grunt shell:compile_dev` to re-compile all templates.
+  - `grunt shell:compile_dev --path=about-us` to re-compile only template(s) which serve about-us path in URL.
+- To fix eslint errors run `npm run eslint`
 
+## Test link deployment
 
-How to work with this project
-=============================
+You can manually deploy your test link using gh-pages with the following configurations:
 
 ### Deploy to your gh-pages for the first time
 
-1. Register your application [here](https://developers.binary.com/applications/). This will give you the ability to redirect back to your github pages after login.
-Use `https://YOUR_GITHUB_USERNAME.github.io/binary-static/en/logged_inws.html` for the Redirect URL and `https://YOUR_GITHUB_USERNAME.github.io/binary-static/en/redirect.html` for the Verification URL.
+1.  Register your application [here](https://developers.binary.com/applications/). This will give you the ability to redirect back to your Github pages after login.
+    Use `https://YOUR_GITHUB_USERNAME.github.io/binary-static/en/logged_inws.html` for the Redirect URL and `https://YOUR_GITHUB_USERNAME.github.io/binary-static/en/redirect.html` for the Verification URL.
 
```

Comment:
'YOUR_GITHUB_USERNAME'  should be replaced with 'YOUR_ORGANMIZATION_NAME' both for Redirect URL and Verification URL